dotcom-rendering/src/lib/pillars.ts

```diff
@@ -134,6 +148,18 @@ export const pillarPalette_DO_NOT_USE: Record<
 		500: specialReport[500],
 		800: specialReport[800],
 	},
+	[ArticleSpecial.SpecialReportAlt]: {
+		dark: palette.specialReportAlt[100],
```

Comment:
This is the new way to refer to palette coming from `source-foundations`. 
See:
https://github.com/guardian/source/blob/main/packages/%40guardian/source-foundations/src/index.ts#L38-L53
https://github.com/guardian/source/blob/main/packages/%40guardian/source-foundations/src/colour/palette.tsx#L217-L222